crud_operation_dict.py

```diff
@@ -0,0 +1,12 @@
+from dict_crud.check.dict import getval
+
+
+getval.set("nawalTwo", "neerajTwo")
+print("--------All_the_data--------")
+print(getval._store_data)
```

Comment:
do you want to directly access the private variable or do you want to expose any API to access this variable values?